bin/git/git.js

```diff
@@ -59,6 +59,11 @@ const TesterFirstCommittedAtQuery = gql`
         commit
         committedAt
       }
+      pullRequest(mergeInfo: { commit: $commit, baseRefName: $branch }) {
+        lastHeadBuild {
+          commit
+        }
+      }
```

Comment:
Did you expose this in `authorizedAppFieldsViaAppCode`?

Reply:
Once I tested this out I did ;)